### PR TITLE
Add feature name in the error reports of feature transformations.

### PR DESCRIPTION
**Pull request by classicsong**
*Issue #, if available:*
Most of error reports of feature transformations do not provide the information of the name of the feature having issues.

*Description of changes:*
Optimize the error reports.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python/graphstorm/gconstruct/transform.py [hunk 1]
```diff
@@ -633,11 +637,13 @@ def call(self, feats):
...
         assert isinstance(feats, (np.ndarray, ExtMemArrayWrapper)), \
-            "Feature of NumericalMinMaxTransform must be numpy array or ExtMemArray"
+            f"Feature {self._feat_name} of NumericalMinMaxTransform " \
+            "must be numpy array or ExtMemArray"
 
         assert not np.any(self._max_val == self._min_val), \
             f"At least one element of Max Val {self._max_val} " \
-            f"and Min Val {self._min_val} is equal. This will cause divide by zero error"
+            f"and Min Val {self._min_val} is equal for feature {self._feat_name}. " \
+            "This will cause divide by zero error"
```
[the remark this answers is not in the document]
Comment (jalencato) [reply python/graphstorm/gconstruct/transform.py:646]: Should here be self._feat_name or self.feat_name, I see the above change and this are in the same function, but refer to the different name.

I think it should be feat_name according to https://github.com/classicsong/graphstorm/blob/11e49900125f21fa959ae472279608df493dab8a/python/graphstorm/gconstruct/transform.py#L195C1-L223C31

**Review by jalencato — COMMENTED** [review]
A little typo here.

File: python/graphstorm/gconstruct/transform.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (classicsong) [reply python/graphstorm/gconstruct/transform.py]: It is a typo.
Updated.

**Review by jalencato — APPROVED** [review]
LGTM

**Pull request merged**